WORKSPACE

```diff
@@ -53,16 +53,16 @@ http_archive(
     strip_prefix = "redis-plus-plus-1.2.3",
     url = "https://github.com/sewenew/redis-plus-plus/archive/refs/tags/1.2.3.zip",
 )
-
-http_archive(
-    name = "rocksdb",
-    build_file = "//build_deps/toolchains/rocksdb:rocksdb.BUILD",
-    sha256 = "2df8f34a44eda182e22cf84dee7a14f17f55d305ff79c06fb3cd1e5f8831e00d",
-    strip_prefix = "rocksdb-6.22.1",
-    urls = [
-        "https://github.com/facebook/rocksdb/archive/refs/tags/v6.22.1.tar.gz",
-    ],
-)
+#
```

Comment:
I cancel the compile and install for RocksDB and copy the headers of Rocksdb  to `tensorflow_recommenders_addons/dynamic_embedding/core/lib/rocksdb/` and request users to guarantee their machine has RocksDB lib.